tests/commons/feature_toggles/feature_toggles_manager_test.py

```diff
@@ -0,0 +1,63 @@
+from commons.feature_toggles.feature_toggle_manager import FeatureToggleManager
+from unittest.mock import Mock, patch
+from pytest import mark
+
+
+def mock_payload(enabled, user):
```

Comment:
Maybe, we could change the name of the function to `mock_feature_toggle_config_response` or something similar. `mock_payload` it seems too general to me.

Reply:
We have changed the name to `mock_feature_toggle_config_response`